tictactoe.py

```diff
@@ -1,26 +1,63 @@
+winningcombo_1 = [1, 2, 3]
+winningcombo_2 = [4, 5, 6]
+winningcombo_3 = [7, 8, 9]
+winningcombo_4 = [1, 4, 7]
+winningcombo_5 = [2, 5, 8]
+winningcombo_6 = [3, 6, 9]
+winningcombo_7 = [1, 5, 9]
+winningcombo_8 = [3, 5, 7]
+
+print("""
+Let’s play tic-tac-toe! The developer is a newbie python coder 🐍
+so for now, plz only type integers from the empty list. 😜
+""")
+
 def game():
-    print("Lets play! Only integers between 1-9 plz:")
-
-    for playermove in range(0,9):
-        playermove = int(input(">>> "))
-
-        if playermove == 1:
-            print("Top left corner, all right. ↖️")
-        if playermove == 2:
-            print("Top center. ⬆️")
-        if playermove == 3:
-            print("Top right corner, nice. ↗️")
-        if playermove == 4:
-            print("Left center. ⬅️")
-        if playermove == 5:
-            print("Yeah that’s a good one. 🆒")
-        if playermove == 6:
-            print("Right center. ➡️")
-        if playermove == 7:
-            print("Bottom left corner, good job. ↙️")
-        if playermove == 8:
-            print("Bottom center. ⬇️")
-        if playermove == 9:
-            print("Bottom right corner, cool. ↘️")
+
+    if len(empty_squares) > 1:
+
+        next_turn_is_x = True
+
+        if next_turn_is_x == True:
+            playermove = int(input("\nPlayer X: "))
+            empty_squares.remove(playermove)
+            playerX_moves.append(playermove)
+            print(">>> POSITIONS by X: ", playerX_moves)
+            print(">>> POSITIONS by 0: ", player0_moves)
+            print(">>> Empty ", empty_squares)
+            next_turn_is_x = False
+
+        if set(winningcombo_1).issubset(playerX_moves) or set(winningcombo_2).issubset(playerX_moves) or set(winningcombo_3).issubset(playerX_moves) or set(winningcombo_4).issubset(playerX_moves) or set(winningcombo_5).issubset(playerX_moves) or set(winningcombo_6).issubset(playerX_moves) or set(winningcombo_7).issubset(playerX_moves) or set(winningcombo_8).issubset(playerX_moves):
+            print(">>> Tic-tac-toe and 🥇  for X")
+            return
+
+        if next_turn_is_x == False:
+            playermove = int(input("\nPlayer 0: "))
+            empty_squares.remove(playermove)
+            player0_moves.append(playermove)
+            print(">>> POSITIONS by X: ", playerX_moves)
+            print(">>> POSITIONS by 0: ", player0_moves)
+            print(">>> Empty ", empty_squares)
+            next_turn_is_x = True
+
+        if set(winningcombo_1).issubset(player0_moves) or set(winningcombo_2).issubset(player0_moves) or set(winningcombo_3).issubset(player0_moves) or set(winningcombo_4).issubset(player0_moves) or set(winningcombo_5).issubset(player0_moves) or set(winningcombo_6).issubset(player0_moves) or set(winningcombo_7).issubset(player0_moves) or set(winningcombo_8).issubset(player0_moves):
+            print(">>> Tic-tac-toe and 🥇  for 0")
+            return
+
+        game()
+
+    elif len(empty_squares) == 1:
```

Comment:
In Python, you can use anything you want as a condition in an `if`, including lists! And an empty list will cause the if to not run. So:

```python
things = ['single thing']
if len(things) > 1: 
    # do the many-things actions
elif things:
    # things can't be empty if we got here. 
    # But len(things) > 1 is also not true, so that must mean there's exactly 1 thing!
else:
    # We are all out of things :-(
```

In general, _I_ prefer to order my `if/elif/else` like this, so I can type less. But the jury's still out on whether that's a good idea or not. But I thought you'd find it helpful that you can do it like that, in case you didn't know!